homeassistant/components/lektrico/config_flow.py

```diff
@@ -0,0 +1,159 @@
+"""Config flow for Lektrico Charging Station."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from lektricowifi import Device, DeviceConnectionError
+import voluptuous as vol
+
+from homeassistant.components.zeroconf import ZeroconfServiceInfo
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import (
+    ATTR_HW_VERSION,
+    ATTR_SERIAL_NUMBER,
+    CONF_FRIENDLY_NAME,
+    CONF_HOST,
+    CONF_TYPE,
+)
+from homeassistant.core import callback
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_FRIENDLY_NAME): str,
```

Comment:
Only helpers are allowed to set friendly names in the config flow. Make sure to set the name to a user recognizable name instead, like a model.

Reply:
I removed CONF_FRIENDLY_NAME and made the changes.